### ISSUE TITLE
Create new clients to parse vgmdb.net instead of using vgmdb.info

### ISSUE THREAD

**Issue by HOZHENWAI** [HOZHENWAI/Beets-Plugin_VGMdb#10]

**Comment by SingularReza** [comment]
Any progress on it yet? vgmdb.info is returning 503 for everything except the home page

**Comment by HOZHENWAI** [comment]
i had good advances on it... kept it on the bad backburner since i had some issues to parse the information that sometimes occured in the notes fields (that vgmdb.info did not parse)... sometimes there are critical information in those notes...
I'll get back on it since vgmdb.info is down... thanks for the info

**Comment by SingularReza** [comment]
You could probably reuse a lot of code as it seems like the issue is vgmdb specifically banning the requests from vgmdb.info domain rather than a problem with the implementation

**Comment by SingularReza** [comment]
Ended up running vgmdb.info on a docker container and pointing the API endpoints to it. Could probably be added to the readme as a temporary solution till the parser gets worked on.

**Comment by HOZHENWAI** [comment]
yeah, that would be a nice thing to add. I'll add it later. For now, it seems vgmdb.info is working again.

**Comment by blitzmikyu** [comment]
Any tips on getting this plugin to work with local docker instance of vgmdb?

I'm using hufman/vgmdb_web:latest image
It seems to query it, I see the requests but beets is just getting network problems. I did modify VGMplug.py to point to the local instance.

**Comment by HOZHENWAI** [comment]
@blitzmikyu what's your modification like? what's the error like? have you made sure the vgmdb_web api is reachable (with a get)?

**Comment by blitzmikyu** [comment]
`
> /lsiopy/lib/python3.12/site-packages/beetsplug/VGMplug.py
`
`
    search_url = "http://10.0.0.226/search/"
    search_albums_url = "http://10.0.0.226/search/albums/"
    album_url = "http://10.0.0.226/album/"
`

Updated these to point to the container running vgmdb_web
![image](https://github.com/user-attachments/assets/13f15830-f2ff-40ea-bcd8-1f3fcbeddd0e)

In the top example, I query for catalog. It returns 2442 album on vgmdb which is the correct response for that ID.

**Comment by HOZHENWAI** [comment]
ok, will try to write the guide after testing... are you also using hufman/vgmdb? what is your docker compose like?

**Comment by blitzmikyu** [comment]
No, just _web. Nothing complicated, I don't have any keys for discog, spotify etc hooked up.

services:
  vgmdb_web:
    image: hufman/vgmdb_web
    ports:
    - 80:80

Service from my browser functions as it should.

**Comment by HOZHENWAI** [comment]
fix is simple, row:396, you have `req = requests.get(f"{self.album_url}{album_id}?format=json")`, replace it with `req = requests.get(f"{self.album_url}{album_id}?format=json", headers={"User-Agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/117.0"})`

**Comment by blitzmikyu** [comment]
Thanks! Seems to be working now with that change.
